pages/en/lb4/Testing-Your-application-advanced.md

```diff
@@ -225,7 +234,21 @@ describe('smoke test', () => {
   });
 });
 ```
-
+<!--
+  FIXME(kev): This misses the fundamental purpose of the "Red" part of Red,
+  Green, Refactor. You don't create a test that will arbitrarily fail, but
+  rather one that will call the appropriate functions/objects under test.
+  (In some languages, like TypeScript, this will require you to create empty
+  scaffolding to avoid compilation errors).
+  
+  The *reason* for this is that you want to make sure your test doesn't pass
+  without any assistance from the code under test. If it **DOES**, then you've
+  successfully demonstrated that your test itself is what's incorrect and you
+  can adjust it before getting too far down the road with the implementation.
+
+  I know this isn't a part of my PR, which is why I'm only leaving this comment
+  instead of fixing it.
+ -->
```

Comment:
when I run `npm start` on local, there is an error complains
```
Liquid Exception: Invalid syntax for include tag: content="src/apidef/swagger.json".js" Valid syntax: {% include file.ext param='value' param2='value' %} in pages/en/lb4/Testing-Your-application.md
jekyll 3.3.1 | Error:  Invalid syntax for include tag:

  content="src/apidef/swagger.json".js"

Valid syntax:

  {% include file.ext param='value' param2='value' %}
```
branch `gh-pages` doesn't fail on it. 
This line is not directly modified in PR, maybe somewhere else affects it?

Reply:
oops I commented in the wrong file that's why I didn't find it, should be
https://github.com/strongloop/loopback.io/pull/475/files#diff-ad241bc342976380fb64dab3b286802bR248
their names are so similar lol